Source/WebCore/Modules/identity/IdentityRequestProvider.h

```diff
@@ -26,14 +26,16 @@
 #pragma once
 
 #include "IdentityCredentialProtocol.h"
-#include <JavaScriptCore/JSObject.h>
-#include <JavaScriptCore/Strong.h>
+#include "OpenID4VPRequest.h"
+#include <variant>
 
 namespace WebCore {
 
+using DigitalCredentialRequest = std::variant<OpenID4VPRequest>;
```

Comment:
Why are we using a variant if there is a single possible type?

Reply:
As you guessed in a later comment, this will need to be a `std::variant`, but I agree that we don't need it right now.

Reply:
Also, I think instead of including `OpenID4VPRequest.h`, you can get away with a forward declaration for `OpenID4VPRequest`, if you do keep the `std::variant` in this PR.

Reply:
Will drop the variant aspect until such time as another variant gets standardized and just set it to OpenID4VPRequest for now.